root_encryption.tf

```diff
@@ -21,6 +21,24 @@ module "s3_external_kms_key" {
   }
 }
 
+module "sns_external_kms_key" {
+  source   = "./da-terraform-modules/kms"
+  key_name = "tdr-sns-external-kms-${local.environment}"
+  tags     = local.common_tags
+  default_policy_variables = {
+    user_roles = concat([
+      module.consignment_export_task_role.role.arn,
+    ], local.aws_sso_export_bucket_access_roles)
```

Comment:
Don't think should use the `local.aws_sso_export_bucket_access_roles` as this is just access to the SNS topic encryption. That group of roles includes "export" which is for digital archivist to get access to the S3 buckets containing the bagit packages. Not sure that is relevant here.

Reply:
Ah ok, I wasn't sure so I guessed. I've taken it out.